code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -563,6 +583,223 @@
 		new /obj/item/circular_saw(src)
 		new /obj/item/surgicaldrill(src)
 
+/obj/item/storage/backpack/duffel/magic_nanny_bag
+	name = "magic nanny bag"
+	desc = "Not to be confused with a magic granny bag. Zip it up to make it unable to be dropped while closed."
+	icon_state = "magic_nanny_bag"
+	item_state = "magic_nanny_bag"
+	max_w_class = WEIGHT_CLASS_HUGE
+	slot_flags = 0
+	storage_slots = 256
+	max_combined_w_class = 256 //get your 8 bit magic bags here. Also it's wizard, at some point this many items will just make it crowded.
+	silent = TRUE
+	zip_time = 0
+	resistance_flags = FIRE_PROOF
+	open_icon_sprite = "magic_nanny_bag_open"
+	antidrop_on_zip = TRUE
+	var/max_value = 7 // make define
```

Comment:
This looks like a todo, either indeed make it a define or autodoc this/give it a better varname

Reply:
Done!